packages/cozy-client/src/associations/Association.js

```diff
@@ -70,15 +70,118 @@
  */
 export default class Association {
   constructor(target, name, doctype, { get, query, mutate, save }) {
+    /**
+     * The document declaring the relationship
+     * @type {object}
+     */
     this.target = target
+    /**
+     * The name of the relationship.
+     * @type {string}
+     * @example 'author'
+     */
     this.name = name
+    /**
+     * Doctype of the relationship
+     * @type {string}
+     * @example 'io.cozy.authors'
+     */
     this.doctype = doctype
+    /**
+     * Method which get the document specified in the relationship.
+     * Typically CozyClient.get
+     * @type {function}
+     */
     this.get = get
+    /**
+     * Method which perform a query to retrieve relationship(s).
+     * Expect a QueryDefinition object as parameter.
+     * Typically CozyClient.query
+     * @type {function}
+     */
     this.query = query
+    /**
+     * Method which perform a mutation on the relationship.
+     * Typically CozyClient.mutate
```

Comment:
I would remove the typically, I find it vague, and it raises too many questions. Does it mean that you can instantiate a relationship with another `mutate` ? What would it mean ?